components/mailbox/src/init.spec.js

```diff
@@ -1280,13 +1265,28 @@ describe("Mailbox Integration: Show draft message in Email thread", () => {
 
   it("Click reply/forward to message in thread opens existing draft", () => {
     cy.get("@mailbox").invoke("prop", "show_reply", true);
+    cy.get("@mailbox").invoke("prop", "show_reply_all", true);
     cy.get("@mailbox").invoke("prop", "show_forward", true);
     cy.get("@mailbox").find(".email-row.condensed").should("have.length", 1);
     cy.get("@mailbox").find(".email-row.condensed").click();
 
     //Click on reply should open existing draft
     cy.get("@email").find(".reply").should("exist");
     cy.get("@email").find(".reply").click();
```

Comment:
Make sure the events `replyClicked`, `replyAllClicked` and `forwardClicked ` are called.